x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -330,7 +320,7 @@ const FIELDS_WITHOUT_CELL_ACTIONS = [
 ];
 
 export function AlertsTableTGrid(props: AlertsTableTGridProps) {
-  const { indexNames, rangeFrom, rangeTo, kuery, workflowStatus, setRefetch, addToQuery } = props;
+  const { indexNames, rangeFrom, rangeTo, kuery, workflowStatus, setRefetch } = props;
```

Comment:
@fkanout We should delete `workflowStatus` from here. There are probably a few more places that workflowStatus might need to be deleted as well. This is not needed, since we are not filtering by workflowStatus anymore in the UI. That's why we still see `workflowStatus:open` in the urlbar when we first load the page. This is not needed anymore

Reply:
Same as here:
https://github.com/elastic/kibana/pull/118723#discussion_r756756041

Reply:
Removing `workflowStatus` It will blocked by `TGridStandaloneCompProps` requires `filterStatus`. Because the value of `filterStatus` is `workflowStatus`

Reply:
@fkanout I summarize what we discussed:

- We agreed to get this PR merged as it is, so that we don't block @ersin-erdal blocked with his PR.
- filterStatus at the moment has value `workflowStatus`, but nothing prevents us by changing it back to alert status. This change was introduced as part of this PR https://github.com/elastic/kibana/pull/109817, so we could pair together to manually revert some workflows status leftover that is not needed
- we agreed to discuss into today's team meeting, how we deal with situations where product requires us to keep some code in our codebase that could possibly be reused in the future, but on the other hand makes the code harder to read
- based on team's feedback as well, we might need to follow up with another PR to clean up the code